src/EditorFeatures/CSharpTest/Recommendations/ThisKeywordRecommenderTests.cs

```diff
@@ -526,5 +526,109 @@ void M()
     {
         var c = new C { x = 2, y = 3, $$");
         }
+
+        [WorkItem(725)]
+        [WorkItem(1107414)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public void InExpressionBodiedMembersProperty()
+        {
+            VerifyKeyword(@"
+class C
+{
+    int x;
+    int M => $$");
+        }
+
+        [WorkItem(725)]
+        [WorkItem(1107414)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public void InExpressionBodiedMembersProperty_1()
+        {
+            VerifyKeyword(@"
+class C
+{
+    int x;
+    int M => $$
```

Comment:
@rchande The `VerifyInFrontOfComment` seems to include a comment but I don't think the tests try to fill the file. In this particular case, the position could be inside a trivia belonging to the leading trivia of the `int` in the next line `int p;`